rusqlite_migration/build.rs

```diff
@@ -0,0 +1,30 @@
+use std::{
+    env,
+    fs::{read_to_string, File},
+    io::{self, BufWriter, Write},
+};
+
+fn main() {
```

Comment:
Maybe make it return `Result<(), Box<dyn Error>>` and get rid of all those `unwrap()`s?

Reply:
You are right, done.